ncat/docs/ncat.xml

```diff
@@ -942,6 +942,104 @@
     file.</para>
   </refsect1>
 
+  <reftsec1 id="ncat-diff">
+    <title> Differences between Ncat and Classical nc </title>
+    <para> There are a few changes and additional features in ncat as
+    compared to classical netcat. The following list highlights them
```

Comment:
Instead of "classical" we can call it "traditional or OpenBSD netcat." And keep the discussion here to those options that are the same in both [traditional](https://manned.org/nc.traditional) and [OpenBSD netcat](http://man.openbsd.org/nc). Where they differ from each other, we do not need to document our choice outside the normal documentation.